AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -221,6 +221,8 @@ eco-area.com#%#//scriptlet("adjust-setInterval", ".stop()", "", "0.02")
 respuestadetarea.com#%#//scriptlet("adjust-setTimeout", "timers", "", "0.02")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/70716
 edurespuestas.com#%#//scriptlet("adjust-setTimeout", "timers", "4000", "0.02")
+! https://github.com/AdguardTeam/AdguardFilters/issues/71591
+bittools.net#%#//scriptlet('adjust-setInterval', 'counter', '1000', '0.02')
```

Comment:
It seems that it's checked on server side and this code is generated after about 15 seconds, so maybe it would be better to change it to something like:
`bittools.net#%#//scriptlet('adjust-setInterval', 'counter', '1000', '0.5')`